components/inputs/test/input-number.test.js

```diff
@@ -368,6 +390,33 @@ describe('d2l-input-number', () => {
 		});
 	});
 
+	describe('value attribute binding', () => {
+
+		[0, 1].forEach((number) => {
+			it(`setting wrapper to ${number} should set input-number value to ${number}`, async() => {
+				const inputWrapper = `<${inputWrapperTag}></${inputWrapperTag}>`;
+				const elem = await fixture(inputWrapper);
+				elem.number = number;
+				await elem.updateComplete;
+				const inputVal = elem.shadowRoot.querySelector('d2l-input-number').value;
+
+				expect(inputVal).to.equal(number);
+			});
+		});
+
+		[undefined, null].forEach((number) => {
```

Comment:
Summary of test results:

Lit 1:
With default number converter: Passes
With custom converter: Passes

Lit 2: 
With default number converter: Fails due to Lit internal attribute conversion changes (sets to 0 instead of undefined)
With custom converter: Passes